lib/tasks/spec.rake

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+if Rake::Task.task_defined?('spec') # only if current environment knows rspec
+  Rake::Task['spec'].actions.clear
+  RSpec::Core::RakeTask.new(:spec) do |t|
+    t.rspec_opts = '--tag ~type:system'
+  end
+  
+  Rake::Task['spec:system'].actions.clear
+  namespace :spec do
+    RSpec::Core::RakeTask.new(:system) do |t|
+      t.pattern = './spec/system/**/*_spec.rb'
+      t.rspec_opts = '--tag type:system'
+    end
+
+    namespace :system do
+      desc 'Prepare frontend before running system tests'
+      task :prepare do
```

Comment:
We can either run the ember server like developing locally or inject the js and css files, as we do in the docker build...